### PR TITLE
Mpdeploy buttons

### PR DESCRIPTION
**Pull request by dietertaotesting**
TAO-1711, TAO-1712
Dependencies
https://github.com/oat-sa/extension-tao-mp/pull/72
https://github.com/oat-sa/extension-tao-testqti/pull/234

### CODE REVIEW

File: models/classes/PreviewConfigClientRegistry.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+use oat\tao\model\ClientLibConfigRegistry;
+
+/**
+ * PreviewClientConfigRegistry
+ *
+ * @author Dieter Raber
+ */
+class PreviewClientConfigRegistry extends ClientLibConfigRegistry
+{
```
Comment (jsconan) [inline models/classes/PreviewConfigClientRegistry.php]: Woops, there is a mistake here: the class name is not the same as the file name

File: models/classes/PreviewConfigClientRegistry.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+
+namespace oat\taoItems\models;
+
```
Comment (jsconan) [inline models/classes/PreviewConfigClientRegistry.php]: This namespace does not comply to the place of this file...

**Comment by dietertaotesting** [comment]
file name and ns are now sorted

File: models/classes/PreviewClientConfigRegistry.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+
+namespace oat\taoItems\model;
+
```
Comment (jsconan) [inline models/classes/PreviewClientConfigRegistry.php]: I fixed the namespace: there is an alias from `extensionName\model` to `extensionName\models\classes`

**Pull request merged**